src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/OffenceEntity.java

```diff
@@ -60,4 +65,11 @@ public class OffenceEntity extends BaseImmutableEntity implements Serializable
 
     @Column(name = "LIST_NO", nullable = false)
     private final Integer listNo;
+
+    @LazyCollection(value = LazyCollectionOption.FALSE)
+    @JsonIgnore
+    @OneToMany(mappedBy = "offence", orphanRemoval=true, cascade = CascadeType.ALL)
+    @OrderColumn(name = "JUDICIAL_RESULTS_ORDER", nullable = false)
```

Comment:
Testing the order column atm, By default hibernate should manage the list order using this column.

Reply:
IT confirms that hibernate maintains the order of insert without any extra logic.